pkg/rpc/params.go

```diff
@@ -60,3 +61,20 @@ func (p Params) ValueAtAndType(index int, valueType string) (*Param, bool) {
 
 	return nil, false
 }
+
+func (p Params) Value(index int) (*Param, *Error) {
+	if len(p) <= index {
+		return nil, newError(-2146233086, http.StatusOK, "Index was out of range. Must be non-negative and less than the size of the collection.\nParameter name: index", "", nil)
```

Comment:
I propose to put this magic error code (and below one) into the constant with mnemonic name (disregarding the fact it is used only once). Then, code would be slightly readable.